Slack.Automation/Promact.Core.Repository/ScrumRepository/ScrumBotRepository.cs

```diff
@@ -383,7 +377,51 @@ public class ScrumBotRepository : IScrumBotRepository
         }
 
 
-
+        /// <summary>
+        /// Used to add channel manually by command "add channel channelname"
+        /// </summary>
+        /// <param name="ChannelName"></param>
+        /// <param name="ChannelId"></param>
+        /// <param name="Username"></param>
+        /// <returns></returns>
+        private async Task<string> AddChannelManually(string ChannelName, string Username, string ChannelId)
+        {
+            var returnMsg = string.Empty;
+            if (ChannelId.StartsWith(StringConstant.GroupNameStartsWith, StringComparison.Ordinal))
+            {
+                // getting user name from user's slack name
+                var applicationUser = _applicationUser.FirstOrDefault(x => x.SlackUserName == Username);
+                // getting access token for that user
+                if (applicationUser != null)
+                {
+                    // get access token of user for promact oauth server
+                    var accessToken = await _attachmentRepository.AccessToken(applicationUser.UserName);
+                    //get the project details of the given channel name
+                    var project = await _projectUser.GetProjectDetails(ChannelName, accessToken);
+                    //add channel details only if the channel has been registered as project in OAuth server
+                    if (project != null && project.Id > 0)
```

Comment:
When no project is fetched. It will return null but second condition is not useful.